### PR TITLE
Upgrade docs and release a new alpha version

### PR DESCRIPTION
**Pull request by pranavmalaviya**
A few minor changes for the alpha version
- Change the long description and release a new version for this
- Disable RHEL job from advanced prechecks
- Add warnings with readmes since the docs can be inaccurate, especially RHEL/Debian

### CODE REVIEW

File: unix-alpha/setup.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 from distutils.core import setup
 
-with open("README.md", "r") as fh:
+with open("alpha-description.md", "r") as fh:
```
Comment (SamDesmondKing) [inline unix-alpha/setup.py:3]: Cool, I take it this is the file that configures the pypi page?

**Review by SamDesmondKing — APPROVED** [review]
Looks good!

File: unix-alpha/setup.py (hunk 1 above)
Comment (pranavmalaviya) [reply unix-alpha/setup.py:3]: You're right. If you see line#12 of this file, we are adding this file in the `long_description` variable there.

File: unix-alpha/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Contributing to _Machine Stats_
 
+| :exclamation:  Warning: This is not a stable release! The following document can be inaccurate. |
```
Comment (jmariomejiap) [inline unix-alpha/CONTRIBUTING.md:3]:
```suggestion
| :exclamation:  Warning: This is not a stable release! The following documention is a work in progress and might not be accurate |
```

File: unix-alpha/README.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 [![PyPI](https://img.shields.io/pypi/v/machine-stats)](https://pypi.org/project/machine-stats/)
 
+| :exclamation:  Warning: This is not a stable release! The following document can be inaccurate. |
```
Comment (jmariomejiap) [inline unix-alpha/README.md:5]:
```suggestion
| :exclamation:  Warning: This is not a stable release! The following documentation is a work in progress and might not be accurate. |
```

**Review by jmariomejiap — APPROVED** [review]
Nice, Thanks @pranavmalaviya 🚀

**Pull request merged**